packages/filesystem/src/browser/filesystem-watcher.ts

```diff
@@ -107,7 +106,9 @@ export class FileSystemWatcher implements Disposable {
     }
 
     protected getIgnored(): Promise<string[]> {
-        const patterns = this.preferences['files.watcherExclude'];
+        /* No need to deep copy as it's only one level deep */
+        const patterns = Object.assign({}, this.preferences['files.watcherExclude']);
```

Comment:
why is this change necessary?

Reply:
Forgot filter returns a new array, removed!